OMCompiler/SimulationRuntime/c/util/simulation_options.c

```diff
@@ -669,6 +687,7 @@ const char *SOLVER_METHOD_NAME[S_MAX] = {
   /* S_IRKSCO */        "irksco",
   /* S_DASSL */         "dassl",
   /* S_IDA */           "ida",
+  /* S_CVODE */         "cvode-experimental",
```

Comment:
Please rename it to `cvode`. You can display a warning or notification to make people aware that this is a new feature. However, changing the name later seems a rather bad idea.

Reply:
Users can't see a warning in OMEdit when choosing the integrator method, only after simulation, and I think changing the solver name is a minor thing.

Reply:
OMEdit shows already a tooltip for the option which labels it as experimental and we can have a notification/warning when simulating. Renaming flags is always a very bad thing.

Reply:
I renamed it